index.html

```diff
@@ -0,0 +1,2059 @@
+              <div class="wrap">
+                <a href="#"
+                  ><img
+                    src="https://d.newsweek.com/en/full/1625634/laura-loomer.jpg?w=397&h=265&q=90&f=92b40e5eafb9a5d800b001b3e28ef3a6"
+                    class="card-img-top"
+                    alt="walmart-store-front"
+                /></a>
+                <a href="#">
+                  <span class="wrap font-weight-bold badge-secondary px-3"
+                    >BUSINESS</span
+                  ></a
+                >
+              </div>
+              <div class="">
+                <h5 class="card-title font-weight-bold pt-3">
+                  <a class="h5-a" href="#"
+                    >CDC Director 'Hopeful' Pandemic Will End by Spring</a
+                  >
+                </h5>
+                <p class="card-text">
+                  Three of the leading vaccine candidates—being developed by
+                  Moderna, Pfizer and AstraZeneca—began large-scale trials
+                  recently.
+                </p>
+              </div>
+            </article>
+            <article class="featured-story card border-0 mb-3">
+              <div class="wrap">
+                <a href="#"
+                  ><img
+                    src="https://d.newsweek.com/en/full/1625579/belarus-protest.jpg?w=397&h=265&q=90&f=850310ffcb3a9a155384e7e4517e09d7"
+                    class="card-img-top"
+                    alt="walmart-store-front"
+                /></a>
+                <a href="#"
+                  ><span class="wrap font-weight-bold badge-secondary px-3"
+                    >BUSINESS</span
+                  ></a
+                >
+              </div>
+              <div class="">
+                <h5 class="card-title font-weight-bold pt-3">
+                  <a class="h5-a" href="#"
+                    >Trump Compares Himself to 'Great Chess Master' in Talks
+                    With Putin, Xi, Kim</a
+                  >
+                </h5>
+                <p class="card-text">
+                  The president spoke of the world leaders being "very sharp"
+                  and "at the top of their game" when discussing the
+                  difficulties of negotiating with them.
+                </p>
+              </div>
+            </article>
+          </div>
+          <!--End of featured stories-->
+          <!--Debate & Opinion section-->
+          <section class="d-md-none gap">
+            <div class="debate py-4">
+              <div class="h2-div font-weight-bold">THE DEBATE</div>
+              <div class="line"></div>
+              <div class="media the-debate">
+                <img
+                  class="d-flex mr-3"
+                  alt="Major"
+                  src="https://d.newsweek.com/en/full/1621698/major-neill-franklin-ret.png?w=150&h=150&f=fedb525be44c92fd2598c72f563e8f95"
+                  style="width: 64px; height: 64px;"
```

Comment:
- [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)